### PR TITLE
Add #gurkcraft-relay support

### PR DESCRIPTION
**Pull request by Akarys42**
This PR adds an automatically updating embed in #gurkcraft-relay, similar to #gurkcraft

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -19,22 +19,8 @@ services:
     init: true
 
     restart: on-failure
-
-    environment:
-      - ENVIRONMENT=DOCKER-DEVELOPMENT
-      - DATABASE_URL=postgres://gurkbotdb:gurkbotdb@postgres:5432/gurkbot
```
Comment (onerandomusername) [inline docker-compose.yaml:25]: These two should probably stay, IIRC they can stay and by defining env_file, it allows overriding these there.

**Review by onerandomusername — CHANGES_REQUESTED** [review]

File: bot/exts/gurkcraft/gurkcraft.py [hunk 2]
```diff
@@ -9,6 +9,9 @@
 
 from bot.constants import Channels, Colours, Minecraft
 
+CHAT_HEADER = "Our gurkan Minecraft server. Join: {Minecraft.server_address} ! \n"
```
Comment (onerandomusername) [inline bot/exts/gurkcraft/gurkcraft.py:12]: f-string

**Review by onerandomusername — APPROVED** [review]
lgtm, tested locally

File: bot/exts/gurkcraft/gurkcraft.py [hunk 3]
```diff
@@ -9,6 +9,9 @@
 
 from bot.constants import Channels, Colours, Minecraft
 
+CHAT_HEADER = f"Our gurkan Minecraft server. Join: {Minecraft.server_address} ! \n"
```
Comment (gustavwilliam) [inline bot/exts/gurkcraft/gurkcraft.py:12]:
```suggestion
CHAT_HEADER = f"Our gurkan Minecraft server. Join: {Minecraft.server_address}! \n"
```
I'm not a fan of the space here. Let's throw it into the sun and watch it go up in silver flames.

File: bot/exts/gurkcraft/gurkcraft.py [hunk 4]
```diff
@@ -55,7 +59,7 @@ async def mcstatus(self, ctx: commands.Context) -> None:
     @tasks.loop(minutes=5)
     async def update_channel_description(self) -> None:
         """Collect information about the server and update the description of the channel."""
-        logger.debug("Updating topic of the #gurkcraft channel")
+        logger.debug("Updating topic of the #gurkcraft and #gurkcraft-relay channel")
```
Comment (gustavwilliam) [inline bot/exts/gurkcraft/gurkcraft.py:62]:
```suggestion
        logger.debug("Updating topic of the #gurkcraft and #gurkcraft-relay channels")
```

File: docker-compose.yaml [hunk 5]
```diff
@@ -19,22 +19,12 @@ services:
...
-      - PREFIX
-      - CHANNEL_DEVLOG
-      - CHANNEL_DEVALERTS
-      - CHANNEL_DEV_GURKBOT
-      - CHANNEL_DEV_REAGURK
-      - CHANNEL_DEV_GURKLANG
-      - CHANNEL_DEV_BRANDING
-      - CHANNEL_LOGS
-      - ROLE_STEERING_COUNCIL
-      - ROLE_MODERATORS
```
Comment (gustavwilliam) [inline docker-compose.yaml:37]: What's going on here?

**Review by gustavwilliam — CHANGES_REQUESTED** [review]
Just some minor wording details.

File: docker-compose.yaml (hunk 5 above)
Comment (Akarys42) [reply docker-compose.yaml:37]: we are just loading everything from `.env` so we don't have to specify vars one by one again

Comment (onerandomusername) [reply docker-compose.yaml:37]: I mentioned this somewhere on discord but can't find where....

**Review by gustavwilliam — APPROVED** [review]
LGTM :shipit:

**Pull request merged**